app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,64 @@
+class QuestionsController < ApplicationController
+  # Callback Methods
+  before_action :set_question, only: [:show, :edit, :update, :destroy]
+
+  # GET /questions
+  def index
+    @active_questions = Question.active.alphabetical
+    @inactive_questions = Question.inactive.alphabetical
+  end
+
+  # GET /questions/:id
+  def show
+  end
+
+  # GET /questions/new
+  def new
+    @question = Question.new
+  end
+
+  # POST /questions
+  def create
+    @question = Question.new(question_params)
+    if @question.save
+      flash[:notice] = "Successfully created #{@question.question}"
+      redirect_to question_path(@question)
+    else
+      render "new"
+    end
+  end
+
+  # GET /questions/:id/edit
+  def edit
+  end
+
+  # PATCH/PUT /questions/:id
+  def update
+    if @question.update(question_params)
+      flash[:notice] = "Successfully updated #{@question.question}"
+      redirect_to question_path(@question)
+    else
+      render "edit"
+    end
+  end
+
+  # DELETE /questions/:id
+  def destroy
+    @question.destroy
+    flash[:notice] = "Successfully deleted #{@question.question}"
+    redirect_to questions_url
```

Comment:
should be questions_path for consistency
my code before was wrong for this and I changed it in the last PR i think